infra/terraform/modules/data_access/gh_webhook_handler/index.py

```diff
@@ -0,0 +1,91 @@
+import hmac
+from hashlib import sha1
+import json
+import os
+
+import boto3
+from botocore.exceptions import ClientError
+
+
+SNS_ARN = os.environ.get("SNS_ARN")
+STAGE = os.environ.get("STAGE")
+GH_HOOK_SECRET = os.environ.get("GH_HOOK_SECRET")
+
+sns = boto3.client("sns")
+
+
+class SNSTopicNotFoundError(Exception):
+    """Raised when trying to publish to a non-existent SNS topic"""
+    pass
+
+
+# e.g. "arn:aws:sns:eu-west-1:1234:dev_gh_team_events"
+def _topic_arn(gh_event):
+    return "{}:{}_gh_{}_events".format(
+        SNS_ARN,
+        STAGE,
+        gh_event,
```

Comment:
It's not a constraint, I tried to not have very long names for other resource names which will include the lambda function name, e.g. `${var.env}_gh_webhook_handler_role_policy`. But I tend to agree with you and be explicit. In theory the limit for resource names in AWS is 256 chars (which I doubt we'll reach).

I suppose 4 chars more would be the end of the world so I'll see if I can change everything to use the full `github` name.